x-pack/plugins/monitoring/server/cluster_alerts/alerts_clusters_aggregation.js

```diff
@@ -9,8 +9,8 @@ import { get, find } from 'lodash';
 import { verifyMonitoringLicense } from './verify_monitoring_license';
 import { i18n } from '@kbn/i18n';
 
-export function alertsClustersAggregation(req, alertsIndex, clusters, checkLicense) {
-  const verification = verifyMonitoringLicense(req.server);
+export async function alertsClustersAggregation(req, alertsIndex, clusters, checkLicense) {
```

Comment:
Same with this one, not used anywhere (except in a test)

Reply:
Agreed, but I didn't want to confuse the purpose of the PR by removing seemingly random files. What do you think about doing this in a separate PR?